src/App.js

```diff
@@ -1,26 +1,70 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import foods from './foods.json'
+import FoodBox from './components/FoodBox';
+import AddButton from './components/AddButton'
+import 'bulma/css/bulma.css'
+import AddForm from './components/AddForm'
+import Search from './components/Search'
+import TodayList from './components/TodayList'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state= {
+      foods: foods,
+      filteredFoods: foods,
+      todayFoods:[],
+      displayForm: false
+    }
+  }
+
+  toggleForm=()=>{
+    let formState = this.state.displayForm
+    formState = !formState
+    this.setState({displayForm: formState})
+  }
+
+  addFood=(event)=>{
+    event.preventDefault();
+    let newFood = {name: event.currentTarget.name.value, calories: event.currentTarget.calories.value, image: event.currentTarget.image.value}
+    this.setState({foods: [...this.state.foods, newFood], filteredFoods: [...this.state.foods, newFood]})
+  }
+
+  searchFood=(event)=>{
+    let greatMatches = this.state.foods.filter((food)=>{return food.name.toLowerCase().startsWith(event.currentTarget.value.toLowerCase())})
```

Comment:
I believe doing only .includes also includes instances where the strings starts with the characters in the input, so doing both filters might be a little redundant. Just using the .includes should work the same.